release/release.go

```diff
@@ -1544,3 +1552,47 @@ func downloadArtifacts(v string, artifactNames []string) {
 		log.Fatalf("expect to download %d artifacts %s, only downloaded %d", len(artifactNames), artifactNames, numArtifactsDownloaded)
 	}
 }
+
+// We want the filenames we record in Papertrail to be unique, but the name of
+// the non-deb/RPM files are things like `release.zip` or `release.msi`. This
+// code will rename these files to something that matches the names that
+// _would_ be used if we were to upload them as official releases.
+//
+// This doesn't _guarantee_ uniqueness since we could run the exact same
+// commit in Evergreen twice, which would generate the same names, but it's
+// close enough. Papertrail will still record the information we give even if
+// we use the same filenames for two different CI runs.
+func renameReleaseFilesForPapetrail() {
+	pf, err := platform.GetFromEnv()
+	if err != nil {
+		log.Fatalf("could not find platform: %v", err)
```

Comment:
It seems the caller function returns an error, should we return the err instead of panic inside the function?

Reply:
Done.

Reply:
I don't think there's much of a difference. If the function returns an error it will ultimately get logged and cause the program to error out.

Reply:
I think there is still some difference? The caller of the function might implement additional logic after the panic error, it's better to return an error instead of panic in place.